Behavioral/Command/Tests/CommandTest.php

```diff
@@ -22,10 +26,13 @@ class CommandTest extends \PHPUnit_Framework_TestCase
      */
     protected $receiver;
 
+    protected $tvReceiver;
+
     protected function setUp()
     {
         $this->invoker = new Invoker();
         $this->receiver = new Receiver();
+        $this->tvReceiver = new TvReceiver();
```

Comment:
This class should not test two seperate examples of a command pattern! Why do we need another example anyway? Is the existing one not clear enough or are there any other reasons to replace it with a new example?